nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1

```diff
@@ -482,8 +482,24 @@ set resizing support (default: true)
 .B fullscreen=<bool>
 start or resume a session in fullscreen mode (default: off)
 .TP 8
-.B keyboard=<string>
-set remote keyboard layout
+.B keyboard=<string> or kbtype=<string>
+
+.BR query | <model>/<layout>
+
+.RS 8
+.TP 8
+.I query
+ask the real X server for the keyboard settings using core protocol
```

Comment:
I did not succeed with query. And I did not fully get the intentions of that code. So I assumed "query" is a value only being used by MacOS clients and only working there (you have told me that MacOs in special in that respect). I do not have (or know) MacOs, so I could not test further. But my tests so far revealed that xkblock is active only when query is used.